gensim/sklearn_integration/sklearn_wrapper_gensim_w2vmodel.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Scikit learn interface for gensim for easy use of gensim with scikit-learn
+Follows scikit-learn API conventions
+"""
+
+import numpy as np
+import six
+from sklearn.base import TransformerMixin, BaseEstimator
+from sklearn.exceptions import NotFittedError
+
+from gensim import models
+from gensim.sklearn_integration import BaseSklearnWrapper
+
+
+class SklW2VModel(BaseSklearnWrapper, TransformerMixin, BaseEstimator):
+    """
+    Base Word2Vec module
+    """
+
+    def __init__(self, size=100, alpha=0.025, window=5, min_count=5,
+            max_vocab_size=None, sample=1e-3, seed=1, workers=3, min_alpha=0.0001,
+            sg=0, hs=0, negative=5, cbow_mean=1, hashfxn=hash, iter=5, null_word=0,
+            trim_rule=None, sorted_vocab=1, batch_words=10000):
```

Comment:
To ensure these values are always kept in sync with the underlying `Word2Vec` class, it might be appropriate to use `inspect.getargspec()` on the `Word2Vec.__init__()` method. Perhaps this method could then just take `**kwargs`, with the defaults dict from `getargspec()` merged together. (Keeping these as a dict might further make many of the explicit attributes unnecessary, and the implementation of `get_params()` trivial.)